allennlp/semparse/contexts/sql_context_utils.py

```diff
@@ -114,7 +117,7 @@ def action_sequence_to_sql(action_sequences: List[str]) -> str:
         if nonterminal == 'statement':
             query.extend(right_hand_side_tokens)
         else:
-            for query_index, token in reversed(list(enumerate(query))):
```

Comment:
We don't need this reversed here now because we traverse them in the reversed order when we generate the action sequence, is that right?

Reply:
Yep, exactly